generator/altitude_generator.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "indexer/feature_altitude.hpp"
+
+#include "std/string.hpp"
+
+namespace routing
+{
+class IAltitudeGetter
```

Comment:
So, do you plan to add "p" prefix to pointer names? "I" to names of variables of integral type? Why this may be interesting for readers? Readers may be interested in what they can do with type, not what the type is. I'm strongly against such Hungarian notation.